PyPDF2/_page.py

```diff
@@ -525,23 +519,8 @@ def mergeRotatedTranslatedPage(
         :param bool expand: Whether the page should be expanded to fit the
             dimensions of the page to be merged.
         """
-
-        translation: List[List[float]] = [[1, 0, 0], [0, 1, 0], [-tx, -ty, 1]]
-        rotation = math.radians(rotation)
-        rotating: List[List[float]] = [
-            [math.cos(rotation), math.sin(rotation), 0],
-            [-math.sin(rotation), math.cos(rotation), 0],
-            [0, 0, 1],
-        ]
-        rtranslation: List[List[float]] = [[1, 0, 0], [0, 1, 0], [tx, ty, 1]]
-        ctm = matrixMultiply(translation, rotating)
-        ctm = matrixMultiply(ctm, rtranslation)
-
-        return self.mergeTransformedPage(
-            page2,
-            [ctm[0][0], ctm[0][1], ctm[1][0], ctm[1][1], ctm[2][0], ctm[2][1]],
-            expand,
-        )
+        op = Transformation().translate(-tx, -ty).rotate(rotation).translate(tx, ty)
```

Comment:
I need to check the results here ... I'm pretty confused by the first translation.